runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -626,12 +641,33 @@ else if (verb.contains("completely"))
 		return "Quest(" + quest + ')';
 	}
 
+	/**
+	 * Parses the Barbarian Assault high gamble reward dialog text into a shortened string for filename usage.
+	 *
+	 * @param text The {@link Widget#getText() text} of the {@link WidgetInfo#DIALOG_SPRITE_TEXT} widget.
+	 * @return Shortened string in the format "(100) Watermelon seed (x50)"
+	 */
+	static String parseBAHighGambleWidget(final String text)
+	{
+		final Matcher highGambleMatch = BA_HIGH_GAMBLE_REWARD_PATTERN.matcher(text);
+		if (highGambleMatch.find())
+		{
+			String reward = highGambleMatch.group(1);
+			String gambleCount = highGambleMatch.group(2);
+
+			return String.format("(%s) %s", gambleCount, reward);
```

Comment:
This should be formatted `High Gamble(count)` like other screenshots. I don't think putting the reward in the filename is valuable.